torch/utils/data/dataloader.py

```diff
@@ -34,6 +43,10 @@ class DataLoader(object):
             at every epoch (default: ``False``).
         sampler (Sampler, optional): defines the strategy to draw samples from
             the dataset. If specified, ``shuffle`` must be False.
+        drop_last (bool, optional): set to ``True`` to drop the last incomplete batch,
```

Comment:
I moved it because it is only used together with `batch_size`, `shuffle` and `sampler`. Grouping them together seemed to make sense to me, but maybe I should respect the arg order more.